docs(nextjs): update README.md to add nextjs info (#136)
davidsonfellipe/fellipe.com: README.md

## 🚀 My Personal Page using Next.js

[![MIT license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat)](https://davidsonfellipe.mit-license.org/)

Welcome to my personal website! This project was developed using Next.js. Feel free to explore the live version of the site at [fellipe.com](https://fellipe.com).

## 🧩 Stack

The website is built on the following technologies:

- React: JavaScript library for building interfaces, providing a component-based architecture.
- Styled-components: This CSS-in-JS library allows an integration of styles with components.

> - Next.js: Leveraging the capabilities of React, Next.js enables server-side rendering, automatic code splitting, and easy deployment.

Author: ```suggestion
- Next.js: Framework that enables server-side rendering (SSR) and static site generation (SSG), automatic code splitting.
```

> ## Hosting

Author: ```suggestion
## 🛜 Hosting
```

The website is hosted on Vercel.

## 💻 Steps to Run Locally

To run the project on your local machine, follow these steps:

1. Clone the repository: `git clone https://github.com/davidsonfellipe/fellipe.com.git`
2. Navigate to the project folder: `cd fellipe.com`
3. Install dependencies: `yarn` (Make sure you have Node.js and yarn installed)
4. Start the development server: `yarn dev`
5. Open your preferred browser and go to [localhost:3000](http://localhost:3000)

## 📖 License

> This code is licensed under the [MIT License](https://davidsonfellipe.mit-license.org), which allows you to use, modify, and distribute it freely while including the original license file.

Author: ```suggestion
This code is licensed under the [MIT License](https://davidsonfellipe.mit-license.org).
```
